app/modules/users/resources.py

```diff
@@ -116,73 +116,15 @@ def patch(self, args, user):
                 db.session,
                 default_error_message="Failed to update user details."
             ):
-            for operation in args:
-                try:
-                    if not self._process_patch_operation(operation, user=user, state=state):
-                        log.info("User patching has ignored unknown operation %s", operation)
-                except ValueError as exception:
-                    abort(code=http_exceptions.Conflict.code, message=str(exception))
-
-            db.session.merge(user)
+            if parameters.PatchUserDetailsParameters.perform_patch(args, user, state):
+                db.session.merge(user)
+            else:
+                abort(
+                    code=http_exceptions.Conflict.code,
+                    message="Failed to update user details."
+                )
```

Comment:
This looks like a code duplication. Let's merge this abort inside `perform_patch`.